core/domain/prod_validators_test.py

```diff
@@ -50,11 +45,11 @@
 
 (
     collection_models, exp_models, feedback_models,
-    story_models, subtopic_models,
+    job_models, story_models, subtopic_models,
```

Comment:
Any specific reason for this change?

Reply:
This file should contain the remaining models ie exploration model
Should I remove all the models except exp_model?